client/assets/scss/colors.scss

```diff
@@ -1,4 +1,9 @@
-$blue: rgba(52, 88, 170, 1);
-$grey: rgba(106, 106, 106, 1);
-$red: rgba(204, 0, 0, 1);
-$white: rgba(255, 255, 255, 1);
\ No newline at end of file
+$darkblue: #0F2F3F;
```

Comment:
```
//blue
$blue-whale: rgba(15, 47, 63);
$blue-whale-75: rgba(15, 47, 63, .75);
$blue-whale-50: rgba(15, 47, 63, .5);
$blue-whale-25: rgba(15, 47, 63, .25);

$chathams-blue: rgba(48, 87, 110);
$chathams-blue-75: rgba(48, 87, 110, .75);
$chathams-blue-50: rgba(48, 87, 110, .5);
$chathams-blue-25: rgba(48, 87, 110, .25);

$denim: rgba(37, 94, 172);
$denim-75: rgba(37, 94, 172, .75);
$denim-50: rgba(37, 94, 172, .5);
$denim-25: rgba(37, 94, 172, .25);

//gold
$marigold: rgba(182, 139, 71);
$marigold-75: rgba(182, 139, 71, .75);
$marigold-50: rgba(182, 139, 71, .5);
$marigold-25: rgba(182, 139, 71, .25);

$metallic-gold: rgba(212, 179, 59);
$metallic-gold-75: rgba(212, 179, 59, .75);
$metallic-gold-50: rgba(212, 179, 59, .5);
$metallic-gold-25: rgba(212, 179, 59, .25);

$marzipan: rgba(233, 202, 130);
$marzipan-75: rgba(233, 202, 130, .75);
$marzipan-50: rgba(233, 202, 130, .5);
$marzipan-25: rgba(233, 202, 130, .25);

//black
$black-russian: rgba(28, 28, 32);
$black-russian-75: rgba(28, 28, 32, .75);
$black-russian-50: rgba(28, 28, 32, .5);
$black-russian-25: rgba(28, 28, 32, .25);

//grey
$masala: rgba(91, 87, 76);
$masala-75: rgba(91, 87, 76, .75);
$masala-50: rgba(91, 87, 76, .5);
$masala-25: rgba(91, 87, 76, .25);

//white
$alabaster: rgba(239, 238, 228);
$alabaster-75: rgba(239, 238, 228, .75);
$alabaster-50: rgba(239, 238, 228, .5);
$alabaster-25: rgba(239, 238, 228, .25);
```
scss have built in functions to lighten and darken or add opacity instead of individually adding different numbers for the opacity.

`rgba($blue-whale, .25)`
`darken( $blue-whale, 10% )`
`lighten( $blue-whale, 10% )`

Reply:
we should also add text, header, etc.. variables, so if we ever decide to change the text color for everything we can just do it here instead of finding it everywhere for `$black-russian`.
`$text-color: $black-russian; `